common/scripted_triggers/wc_coa_triggers.txt

```diff
@@ -22,6 +22,18 @@ european_pagan_coa_trigger = {
 		like_baltic_religion_trigger = yes
 		like_finno_ugric_religion_trigger = yes
 		like_magyar_religion_trigger = yes
+		
+		#Warcraft
+		scope:culture = {
+			OR = {
+				AND = { # Dryads and Night Elves
+					has_graphical_culture = druidic_coa_gfx
+					has_graphical_culture = elunic_coa_gfx
+				}
```

Comment:
If they're supposed to be used together, why you didn't create one graphical culture?

Reply:
The idea behind separating druidic and elunic coas is for situations where there is one without the other. 

Examples would include:
- highborne/nightborne 
- Non-Elunic Wild God worship; goldrinn and agamagan, etc.
- Royal Court hybrid cultures: Druidic tauren/Elunic gilneans. 

In terms of CoA motifs, Elunic imagery features weapon and moon motifs whereas druidic is all animals and trees.

Reply:
But you fixed it.